src/message.js

```diff
@@ -27,8 +27,8 @@ class Message {
     this.meta = meta || {}
     this.conversation_id = [
       this.meta.team_id,
-      this.meta.channel_id,
-      this.meta.user_id || this.meta.bot_id
+      this.meta.channel_id || 'nochannel',
```

Comment:
Only advantage I thought of was readability in case you were logging these or inspecting them.